src/app/footer/footer.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="container footer" 
+    fxLayout="row wrap" 
+    fxLayout.lt-md="column" 
+    fxLayoutAlign="center center" 
+    fxLayoutGap="10px">
+
+  <div fxFlex="100" fxFlex.gt-sm="45" fxLayoutAlign="center">
+    <div style="margin-left: auto; margin-right: auto;">
+      <h4>Our Address</h4>
+        <address>
+          121, Clear Water Bay Road<br> Clear Water Bay, Kowloon<br> HONG KONG<br>
+          <i class="fa fa-phone"></i>: +852 1234 5678<br>
+          <i class="fa fa-fax"></i>: +852 8765 4321<br>
+          <i class="fa fa-envelope"></i>:
+          <a href="mailto:confusion@food.net">confusion@food.net</a>
+        </address>
+    </div>        
+  </div>
+  <div fxFlex="100" fxFlex.gt-sm="45">
+    <div style="text-align:center">
+      <a mat-icon-button class="btn-google-plus" href="http://google.com/+"><i class="fa fa-google-plus fa-lg"></i></a>
```

Comment:
parts of application that are actually dependencies better to extract to components. Here `<i class="fa fa-google-plus fa-lg"></i>` anf other font-awesome deps better to extract to custom components, f.e. `app-large-plus`

1. if we need that icon in other place - no code duplication
2. if any changes due to font-awesome they will be applied in one place

sample code
```
@Component({
    template: `<i class="{className}"></i>`
})
class abstract BaseFontAwesomeComponent {
    constructor(className: string) {
        this.className = `fa ${className} ${this.isLg ? 'fa-lg' : ''}`;
    }

    @Input isLg: boolean;
    className= '';
}

@Component({
    selector: 'app-google-plus-icon'
})
class GooglePlusIcon extends BaseFontAwesomeComponent {
    constructor() {
        super('fa-google-plus')
    }
}

<app-google-plus-icon isLg color={Colors.Orange}/>
```

Reply:
done